src/Modules/OrchardCore.Commerce.Payment/Services/PaymentService.cs

```diff
@@ -198,6 +198,21 @@ await _paymentProvidersLazy
             orderPart.LineItems.SetItems(lineItems);
             orderPart.Status.Text = OrderStatuses.Pending.HtmlClassify();
 
+            // When billing and shipping addresses should match, if an address field is null but the other one is not,
+            // fill out the null field's data.
```

Comment:
Could you rephrase this comment to describe the situation where this is necessary? Currently it feels more like it's just describing the code below, which is not that useful.